src/routes/api/trips.route.js

```diff
@@ -17,7 +14,7 @@ const router = express.Router();
 
 router
   .post('/request', authenticate, validateRequestType, validate(onewayCheck(), 'one-way'),
-    onewayValidateInput, validate(multicityCheck(), 'Multi-city'), multicityValidateInput, tripRequest)
+    onewayValidateInput, validate(returnTripCheck(), 'Return-Trip'), returnTripValidateInput, validate(multicityCheck(), 'Multi-city'), multicityValidateInput, tripRequest)
```

Comment:
Excellent work. However, Instead of disabling eslint, maybe you can move some to the next line, so it can be easier to read.